docs/examples/basic.md

```diff
@@ -0,0 +1,88 @@
+# Basic OTG Configurations
+
+## Single TCP Traffic Flow between two Test Ports
+
+File: [`flow.otg.yaml`](flow.otg.yaml).
+
+This single flow configuration specifies:
+
+* Flow Name: `f1`
+* Packet Schedule: 
+    * `1000` packets 
+    * of `512B` size 
+    * to be transmitted as fixed rate of `100pps`
+* Flow Template: 
```

Comment:
`Packet Template` instead of `Flow Template`?

Reply:
fixed